src/main/java/com/jmb/springfactory/model/dto/PageDto.java

```diff
@@ -0,0 +1,12 @@
+package com.jmb.springfactory.model.dto;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class PageDto<T extends BaseDto> {
+    private List<T> list;
+    private Integer page;
+    private Integer totalPages;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "totalPages" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)